### ISSUE TITLE
OutdatedDocumentation fails when parameter is marked as private val

### ISSUE THREAD

**Issue by matejdro** [detekt/detekt#4365]
## Expected Behavior

Following code should pass:

```kotlin
/**
 * @param myParam Description of param
 */
class MyClass(private val myParam: String)
```

Using either `@param` or `@property` fails.

## Observed Behavior

`OutdatedDocumentation` rule fails on the above code.

## Steps to Reproduce

1. Run detekt on above code
2. Observe results

## Your Environment

* Version of detekt used: 1.19.0
* Version of Gradle used (if applicable): 7.3.1

**Comment by severn-everett** [comment]
I'm unable to reproduce this when using the `@property` doc tag on `private val myParam`. Could you provide a direct reproduction case for this?

**Comment by matejdro** [comment]
Sorry, I cannot reproduce it anymore. My guess is that `UnusedPrivateMember` triggered and I mixed it up with the other warning.
